Makefile

```diff
@@ -26,11 +34,17 @@ qa: bin
 	scripts/qa/vet-high-confidence.sh
 	# qa-post-build steps minus stack tests
 	scripts/qa/docs.sh
+
 all:
 	$(MAKE) bin
 	$(MAKE) docker
 	$(MAKE) qa
 
+debug:
+	$(MAKE) bin-debug
+	$(MAKE) docker-debug
+	${MAKE} qa-debug
```

Comment:
the last `MAKE` has `{}` instead of `()`. is that intentional?

Reply:
No, that's a mistake